.github/workflows/ci.yml

```diff
@@ -75,7 +75,7 @@ jobs:
         run: |
           go mod download
           go mod vendor
-          go generate
+          go generate -mod=vendor
```

Comment:
This _should_ fix the "inconsistent vendoring" build error that the CI was running into.